ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m

```diff
@@ -123,21 +172,17 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
                                                                  mediaType:AVMediaTypeVideo
                                                                   position:AVCaptureDevicePositionUnspecified];
     for (AVCaptureDevice *device in videoevicesSession.devices) {
-        NSString *position = @"unknown";
+        NSString *position = @"";
         if (device.position == AVCaptureDevicePositionBack) {
             position = @"environment";
         } else if (device.position == AVCaptureDevicePositionFront) {
             position = @"front";
         }
-        NSString *label = @"Unknown video device";
```

Comment:
Please don't undo this, I've seen crashes because it was null.

Reply:
reverted